docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Nice work and detailed description. Thanks</p><p>Possible to add a figure to explain the general remapping idea? two grids with a value mapped from one to another.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>